app/views/conferences/_notification_section.html.haml

```diff
@@ -16,7 +16,7 @@
           = link_to e.title, e
     %p
     - if noticket.count == e.count
-      =button_to('No events with tickets', {}, { disabled: true, class: "btn disabled"})
+      =button_to(t('notifications_module.no_event_tickets'), {}, { disabled: true, class: "btn disabled"})
     - else
       =button_to( 'Send ' + name + ' notifications to ' + pluralize( speakers, 'speaker') + ' in ' + pluralize( e.count - noticket.count, 'event'), { action: 'send_notification', notification: state }, { class: "btn danger", data: { confirm: 'Are you sure you want to send ' + name + ' notifications for ' + pluralize(e.count - noticket.count, 'event') +'?', disable_with: 'working...' }})
-  
+      
```

Comment:
trailing space be gone! ;)

Reply:
yep, and an untranslated string above :/

Reply:
Thanks for your review, it's done !